app/src/main/java/com/github/weg_li_android/MainActivity.kt

```diff
@@ -66,4 +265,163 @@ class MainActivity : AppCompatActivity() {
             violationSpinner.adapter = adapter
         }
     }
+
+    override fun onItemClick(view: View?, position: Int) {
+        if(mActionMode != null) {
```

Comment:
In order to avoid indentation, the "return early"-pattern should be used here and at other places. For reference, see https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement